docs/content/manual/volumes/1334-evict-disks-check-vms.md

```diff
@@ -2,20 +2,34 @@
 title: Verify that VMs stay up when disks are evicted
 ---
 
-* Related issues: [#1334](https://github.com/harvester/harvester/issues/1334) Volumes fail with Scheduling Failure after evicting disc on multi-disc node
+* Related issues
+    - [#1334](https://github.com/harvester/harvester/issues/1334) Volumes fail with Scheduling Failure after evicting disc on multi-disc node
+    - [#5307](https://github.com/harvester/harvester/issues/5307) Replicas should be evicted and rescheduled to other disks before removing extrak disk
 
 ## Verification Steps
 
-1. Created 3 node Harvester setup with ipxe example in KVM/libvirt
-1. Added formatted disk to node0 VM
-1. Created three VMs on node0
-1. Created large files on three VMs to see where they were located with dd if=/dev/urandom of=file1.txt count=5192 bs=1M
-1. Checked Longhorn to be sure that some VMs were on new disk
-1. Deleted disk from Harvester
-1. Checked Longhorn to be sure that disk was marked for eviction
-1. Verified that VMs were still available while evicting replicas by running commands from serial console/SSH
-1. Verified that disk was removed from Longhorn and VMS were still up.
+1. Created 3 nodes Harvester.
+1. Added formatted disk (called disk A) to node0 VM in the harvester node page.
+1. Added disk tag `test` on following disk in the longhorn page.
+    1. disk A of node0
+    1. root disk of node1
+    1. root disk of node2
+1. Created storage class with disk tag `test`.
+1. Created volume (called B) with previous storage class. You should check scheduling status in the longhorn page.
+1. Created VM with volume B as extra volume (not boot volume).
+1. The node scheduling status should look like this. {{< image "images/volumes/1334-image-01.png" >}}
+1. Deleted disk A in the harvester node page, and it's not allowed.
+1. Remeber add disk tag `test` to **root disk of node0 before eviection**, otherwise there is no place for replica rescheduling in next step.
```

Comment:
> ~Remeber a~Add disk tag `test` to **root disk of node0**. ~**before eviection**, otherwise there is no place for replica rescheduling in next step.~